src/test/resources/Homework20NG.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework20NG">
+    <parameter name="BaseURL" value="https://bbb.testpro.io"/>
+    <test name="Homework20" preserve-order="false">
+        <classes>
+            <class name="BaseTest"/>
+            <class name="Homework20">
+                <methods>
+                    <include name="playSongTest"/>
+                </methods>
+            </class>
+        </classes>
+    </test>
+    <test name="Homework20" preserve-order="false">
+        <classes>
+            <class name="BaseTest"/>
+            <class name="Homework20">
+                <methods>
+                    <include name="addSongToPlaylist"/>
+                </methods>
+            </class>
+        </classes>
+    </test><test name="Homework20" preserve-order="false">
+    <classes>
+        <class name="BaseTest"/>
+        <class name="Homework20">
+            <methods>
+                <include name="deleteCreatedPlaylist"/>
+            </methods>
+        </class>
+    </classes>
+</test>
+</suite>
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/222947542-61111df3-26d7-4f51-9054-99217fcaeb0b.png)